world.js

```diff
@@ -18,6 +18,20 @@ import {worldMapName, appsMapName, playersMapName} from './constants.js';
 import {playersManager} from './players-manager.js';
 import * as metaverseModules from './metaverse-modules.js';
 
+setTimeout(()=>{
+  const effectApp = metaversefileApi.createApp();
+  (async () => {
+    await metaverseModules.waitForLoad();
+    const {modules} = metaversefileApi.useDefaultModules();
+    const m = modules['specialEffect'];
+    await effectApp.addModule(m);
+
+    effectApp.position.set(-2,2,-2)
+    effectApp.updateMatrixWorld();
+  })();
+
+  scene.add(effectApp);
```

Comment:
We generally should not add things to the scene unless we are an app. And this is not an app, this is the world bootstrap.

Reply:
Yea shouldn't be run by default. I'll adjust the PR, I was eager to show something working, but these changes are inappropriate for master